data-access/Helpers/Licence.cs

```diff
@@ -1,5 +1,3 @@
-using SQE.SqeHttpApi.DataAccess.Models;
```

Comment:
Licence is hurting my eyes. I understand it's correct, but still - hurting my eyes. Don't change it, I'll get better glasses instead.